backend/windmill-queue/src/schedule.rs

```diff
@@ -27,8 +27,10 @@ pub async fn push_scheduled_job<'c>(
         .map_err(|e| error::Error::BadRequest(e.to_string()))?;
 
     let offset = Duration::minutes(schedule.offset_.into());
-    let now = now_from_db(&mut tx).await?;
```

Comment:
you want to have the time of the db since the jobs are scheduled based on the time of the db. There might be a drift between the 2